tests/features/diff_io/CMakeLists.txt

```diff
@@ -9,6 +9,19 @@ function(diff_io_test name)
 endfunction()
 
 diff_io_test(obuftds)
-diff_io_test(obufds)
-diff_io_test(ibufds)
 diff_io_test(iobufds)
+
+add_generic_test(
+    name ibufds
+    board_list basys3 zcu104
+    constr_prefix ibufds
+    sources ibufds.v
+    testbench ibufds_tb.v
+)
+add_generic_test(
+    name obufds
+    board_list basys3 zcu104
+    constr_prefix obufds
+    sources obufds.v
+    testbench obufds_tb.v
+)
```

Comment:
You may improve the `diff_io_test` to take in also the boards list on which to run the test and leave line 12 and 13

Reply:
Done.